packages/node_modules/@node-red/nodes/core/function/89-delay.html

```diff
@@ -114,7 +114,8 @@
             timeout: {value:"5", required:true, validate:function(v) { return RED.validators.number(v) && (v >= 0); }},
             timeoutUnits: {value:"seconds"},
             rate: {value:"1", required:true, validate:function(v) { return RED.validators.number(v) && (v >= 0); }},
-            nbRateUnits: {value:"1", required:false, validate:RED.validators.regex(/\d+|/)},
+            nbRateUnits: {value:"1", required:false,
+                          validate:function(v) { return RED.validators.number(v) && (v >= 0); }},
```

Comment:
```suggestion
            nbRateUnits: {value:"1", required:false, validate:RED.validators.regex(/^(\d+|)$/)},
```

As mentioned in my comment - this is the alternative I would suggest that keeps the original intention of the validator.